client/stylesheets/_toCat.scss

```diff
@@ -3385,7 +3397,7 @@ On click the images will be displayed at normal size to complete the effect
     width: 45%;
 }
 
-.listingItemDetailLeft ul li:nth-child(2), .listingItemDetailRight ul li:nth-child(2) {
+.listingItemDetailLeft ul li:nth-child(2), .listingItemDetailLeft ul li:nth-child(3), .listingItemDetailRight ul li:nth-child(2), .listingItemDetailRight ul li:nth-child(3) {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `listingItemDetailLeft` should be written in lowercase with hyphens<br>Selector `listingItemDetailRight` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line